api/questions/index.js

```diff
@@ -125,201 +122,202 @@ function formatAnswer(body) {
             answer_text: body.text || 'foo bar',
             // answer_state: body.state || 'answer_drafted'
             answer_state: body.state || 'answer_submitted'
-        };
+        }
     }
     return {
         answer_text: 'foo bar',
         // answer_state: 'answer_drafted'
         answer_state: 'answer_submitted'
-    };
+    }
 }
 
 function getAllQuestionsByCase(caseId, userId, options, jurisdiction) {
     return cohCor.getHearingByCase(caseId, options)
         .then(hearing => hearing.online_hearings[0] ? hearing.online_hearings[0].online_hearing_id : createHearing(caseId, userId, options, jurisdiction))
         .then(hearingId => cohCor.getAllRounds(hearingId, options))
-        .then(rounds => rounds && formatRounds(rounds.question_rounds));
+        .then(rounds => rounds && formatRounds(rounds.question_rounds))
 }
 
-
 function getOptions(req) {
     return {
         headers: {
             Authorization: `Bearer ${req.auth.token}`,
             ServiceAuthorization: req.headers.ServiceAuthorization
         }
-    };
+    }
 }
 
 module.exports = app => {
-    const route = express.Router({ mergeParams: true });
-    app.use('/case', route);
+    const route = express.Router({ mergeParams: true })
+    // TODO: we need to put this back to '/case' in the future (rather than '/caseQ') when it doesn't clash with case/index.js
+    app.use('/caseQ', route)
 
     // GET A Question
     route.get('/:case_id/questions/:question_id', (req, res, next) => {
-        const caseId = req.params.case_id;
-        const questionId = req.params.question_id;
-        const options = getOptions(req);
+        console.log('question')
```

Comment:
Not sure if this console.log is required.

Reply:
will remove asap